analysed-packages/kmod/version-v29/kmod-v29-SPDX2TV.spdx

```diff
@@ -0,0 +1,7184 @@
+by the Free Software Foundation. python-kmod is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; withouteven the implied warranty of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details. You should have received a copy of the GNU Lesser General Public License along with python-kmod. If not, see <http://www.gnu.org/licenses/>.
+
+Thus, we concluded LGPL-2.1-only </text>
+LicenseInfoInFile: LicenseRef-LGPL-2.1-only
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: <text> Copyright (C) 2012 W. Trevor King <wking@tremily.us> </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/libkmod/python/kmod/list.pyx
+SPDXID: SPDXRef-item522296
+FileChecksum: SHA1: a7c429f1ed5786fdcae1a2cd0fedb2209b437d83
+FileChecksum: SHA256: fea4a8ee148d63046590940987cb0b7a70c77ae4ebd79eac1d0271d71b0e140e
+FileChecksum: MD5: 1d32c7ea43b09d907788675f4c232eeb
+LicenseConcluded: LicenseRef-LGPL-2.1-only
+LicenseComments: <text> The information in the file is:
+
+This file is part of python-kmod. Python-kmod is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License version 2.1 as published
+by the Free Software Foundation. python-kmod is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; withouteven the implied warranty of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details. You should have received a copy of the GNU Lesser General Public License along with python-kmod. If not, see <http://www.gnu.org/licenses/>.
+
+Thus, we concluded LGPL-2.1-only </text>
+LicenseInfoInFile: LicenseRef-LGPL-2.1-only
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: <text> Copyright (C) 2012 W. Trevor King <wking@tremily.us> </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/libkmod/COPYING
+SPDXID: SPDXRef-item522286
+FileChecksum: SHA1: 545f380fb332eb41236596500913ff8d582e3ead
+FileChecksum: SHA256: 6095e9ffa777dd22839f7801aa845b31c9ed07f3d6bf8a26dc5d2dec8ccc0ef3
+FileChecksum: MD5: a6f89e2100d9b6cdffcea4f398e37343
+LicenseConcluded: LicenseRef-License-of-GNU-Licenses
+LicenseComments: <text> The file contains the text of the LGPL-2.1-or-later. The text of the license is licensed under:
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+Thus we concluded License-of-GNU-Licenses as license of this file. </text>
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-only
+LicenseInfoInFile: LicenseRef-License-of-GNU-Licenses
+LicenseInfoInFile: LicenseRef-LGPL-2.1
+FileCopyrightText: <text> Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+written by James Random Hacker. </text>
```

Comment:
Please remove this, this is not a real copyright statement. For further details please see my comment in the disclosure document